### ISSUE TITLE
coding standard should not recommend putting a "|" character in front of code samples

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5165]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#5165|
|Type|enhancement|
|Created|2011-06-28 05:10:58Z|

This practice is documented to help Emacs users by removing items from the IM-Python menu, but that doesn't strike me as much of an inconvenience, and it screws up all kinds of code formatting tools.

Attachments:

* [emacs-epytext.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ca2/ca278c46bc2f78479593313d0d2998b96283b520/71563ba7883dc90cf68f62da3834defdc14f6f48.patch) (1691 bytes) - added by thijs on 2011-10-03 20:42:47Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5165 5165
type__enhancement enhancement
reporter__glyph glyph
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1309237858000000 1309237858000000
changetime__1318008469000000 1318008469000000
version__None None
owner__thijs thijs
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

Attached patch removes the recommendation (should be merged with a .misc file). There were 2 notes for emacs users, I removed the one related to this ticket.

**Comment by twisted-trac** [comment]
|[<img alt="bitglue's avatar" src="https://avatars.githubusercontent.com/u/1232472?s=50" width="50" height="50">](https://github.com/bitglue)<a name="note_2"></a>|@bitglue set owner to @thijstriemstra|
|-|-|

+1

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [32802]) Apply emacs-epytext.patch: Remove the coding standard recommendation for emacs users to put a '|' character in front of code samples, because it screws up all kinds of code formatting tools.

Author: thijs
Reviewer: indigo
Fixes: [#5165](https://github.com/twisted/twisted/issues/5165)
